### PR TITLE
Add in AppVeyor configuration

### PR DESCRIPTION
**Pull request by jamieconnolly**
This change is [Reviewable](https://reviewable.io/reviews/atomlinter/linter-scss-lint/158)

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  - gem update --system
+  - gem install scss_lint --no-document
+
+before_test:
+  - apm --version
+  - atom --version
+  - ruby --version
+  - gem --version
+  - scss-lint --version
+test_script:
```
Comment (Arcanemagus) [inline appveyor.yml]: `test_script` intuitively _should_ work, but it wasn't for me on another project, if it works here for you though then awesome :stuck_out_tongue:.

File: appveyor.yml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+platform: x64
+
+environment:
+  ruby_version: 22-x64
+  matrix:
+    - ATOM_CHANNEL: stable
+    - ATOM_CHANNEL: beta
+
+install:
+  - appveyor DownloadFile https://atom.io/download/windows -FileName AtomSetup.exe
```
Comment (Arcanemagus) [inline appveyor.yml]: Check out the current [atom/ci](https://github.com/atom/ci/blob/94f7c0cc554826302b8073979f8ae8f9f89661ec/appveyor.yml#L20-L24) version, it uses a much more stable method of getting Atom installed in the CI environment, and on the plus side also runs your configured linters on AppVeyor as well.

File: appveyor.yml [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+  - cd %APPVEYOR_BUILD_FOLDER%
+  - SET PATH=%LOCALAPPDATA%\atom\bin;%PATH%
+  - apm clean
+  - apm install
+  - SET PATH=C:\Ruby%ruby_version%\bin;%PATH%
+  - gem update --system
+  - gem install scss_lint --no-document
+
+before_test:
+  - apm --version
```
Comment (Arcanemagus) [inline appveyor.yml]: If you move to the atom/ci method as described above this line and the next one are done inside that script.

**Comment by jamieconnolly** [comment]
I've updated the configuration with your suggestions and it all passes! Thanks for your input!

**Comment by jamieconnolly** [comment]
I don't think the webhook has been added to the repo. Could you double check @Arcanemagus? Thanks in advance!

**Comment by Arcanemagus** [comment]
AppVeyor is stupid and doesn't do organization level web hooks, so just add it for yourself and it will hook in here.

**Comment by jamieconnolly** [comment]
I've added it, but I had to manually start the build off of master hence why I'm not sure. I don't have access to add hooks to this repo.

**Comment by Arcanemagus** [comment]
Hmmm, looks like _nobody_ has explicit admin access on this repo... I'll just go ahead and add you.

**Comment by Arcanemagus** [comment]
There, you should have admin access on here now, don't mess anything up :stuck_out_tongue:.

**Comment by jamieconnolly** [comment]
Haha I'll try not to. With great power comes great responsibility after all!

**Pull request merged**